docs/example.rst

```diff
@@ -0,0 +1,35 @@
+Example
+=======
+
+There is an example app in `Django Public Admin repository <https://github.com/cuducos/django-public-admin/>`_, inside the ``example/`` directory. This example is meant to be a straightforward use case, having *Django's native admin* running in parallel with *Django Public Admin*.
+
+Requirements
+------------
+
+* Git
+* Python 3.6 or newer with `Poetry <https://python-poetry.org>`_ (or other `PEP 517 <https://www.python.org/dev/peps/pep-0517/>`_ `pyproject.toml` compatible tool)
+
+Running the example
+-------------------
+
+First, clone the repository and install the dependencies:
+
+::
+
+    git clone https://github.com/cuducos/django-public-admin.git
+    poetry install
+
+Then start the application:
+
+::
+
+    poetry run python manage.py runexample
+
+The ``runexample`` command is a wrapper around Django's native ``runserver``. It creates a temporary SQLite database, run migrations, creates a superuser, and collects static files *automagically* before spinning up the development server. If you are having trouble with this command, you can try to delete all these temporary files with ``poetry run python manage.py cleanexample``.
```

Comment:
I tried to stop the server using Control+C and got the error:
```
    return database_name == ':memory:' or 'mode=memory' in database_name
TypeError: argument of type 'PosixPath' is not iterable
```
Same when I tried to run this command.

Reply:
Wow… that's bizarre. Can you share the whole traceback so I can have a better idea about what is causing that? I couldn't replicate this error here : (

Reply:
Just googled it. Looks like it's a Django bug to be fixed in 3.1.1 https://github.com/django/django/commit/77aa74cb70dd85497dbade6bc0f394aa41e88c94